admin-part/src/main/kotlin/CoverageTools.kt

```diff
@@ -84,7 +85,10 @@ internal fun Map<TypedTest, BundleCounter>.methodsCoveredByTest(
             newMethods = changes.newMethods.methods,
             modifiedMethods = changes.allModifiedMethods.methods,
             unaffectedMethods = changes.unaffectedMethods.methods
-        ).also { cache?.set(bundle.hashCode(), it) }
+        ).also {
+            if (finalizedTests?.contains(typedTest) == true)
```

Comment:
Replace to "in"

Reply:
```
            if (finalizedTests != null) {
                if (typedTest in finalizedTests)
                    cache?.set(typedTest, it)
            }
```
I don't think that such code will look better

Reply:
See previous reply